src/utils/trusts.ts

```diff
@@ -213,10 +222,17 @@ const getIndividualTrustee = (
 const getFormerTrusteesFromTrust = (
   appData: ApplicationData,
   trustId?: string,
+  isReview?: boolean,
 ): TrustHistoricalBeneficialOwner[] => {
   let formerTrustees: TrustHistoricalBeneficialOwner[] = [];
+  let trustList: Trust[] | undefined;
+  if (isReview) {
+    trustList = appData[UpdateKey]?.[ReviewTrustKey];
+  } else {
+    trustList = appData[TrustKey];
+  }
   if (trustId) {
-    formerTrustees = appData[TrustKey]?.find(trust =>
+    formerTrustees = trustList?.find(trust =>
       trust?.trust_id === trustId)?.HISTORICAL_BO as TrustHistoricalBeneficialOwner[];
```

Comment:
like above, "as" is wrong & not need.

Reply:
Christy hasn't added this this, as above this is existing code